src/controller/gameController.ts

```diff
@@ -1,47 +1,85 @@
 import { AuthRequest } from './authenticationController.js';
 import {Response, NextFunction} from 'express';
 import { AppError, errorWrapper } from '../utils/errors.js';
+import { v4 as uuidv4 } from 'uuid';
 
 import { LocalStorage } from '../LocalStorage/localStorage.js';
+import { em } from '../database.js';
+import { Game, Player, User } from '../entity/index.js';
+import { Card } from '../LocalStorage/Card.js';
+import { InviteResponse, InviteType } from '../LocalStorage/InviteResponse.js';
+import assert from 'assert';
 import { NopeSocket } from '../router/socketRouter.js';
 
-export const gameInvite =  errorWrapper(async (req: AuthRequest, res:Response, next:NextFunction) => {
-    // 1) Convert the input to ConnectedUser Objects
+export const gameInvite = errorWrapper(async (req: AuthRequest, res:Response, next:NextFunction) => {
+    // 1) Convert the input to a Game Objects
     if (!req.body.players || !Array.isArray(req.body.players)) {
         return next(new AppError('Please provide an array of players', 400));
     }
-
     if (req.body.players.length < 2) {
         return next(new AppError('Please provide at least two players', 400));
     }
 
-    // 2) Check if the players are available
-    const convertedSocket : NopeSocket[] = convertToSocket(req.body.players);
-
+    const game = new Game(uuidv4(), Card.getRandomCard());
+    const userRepository = em().getRepository(User);
+    for (let playerIndex = 0; playerIndex < req.body.players.length; playerIndex++) {

```

Comment:
Hier könnte man einen ForEach Loop benutzen. In der Callback der Foreach loop kann man dann den `PlayerIndex` verwenden